src/backends/odbc/vector-use-type.cpp

```diff
@@ -181,19 +181,39 @@ void* odbc_vector_use_type_backend::prepare_for_bind(SQLUINTEGER &size,
 
             prepare_indicators(vsize);
 
-            size = sizeof(char) * 2;
+#ifdef SOCI_ODBC_WIDE
+            size = sizeof(SQLWCHAR) * 2; // 1 wchar + 1 null terminator
+#else
+            size = sizeof(char) * 2; // 1 char + 1 null terminator
+#endif // SOCI_ODBC_WIDE
+
```

Comment:
Can't it be written just as

```suggestion
            size = sizeof(SQLTCHAR) * 2; // 1 wchar + 1 null terminator
```

?